src/framework/uicomponents/view/popupwindow/popupwindow_qquickview.cpp

```diff
@@ -75,7 +75,11 @@ void PopupWindow_QQuickView::init(QQmlEngine* engine, bool isDialogMode, bool is
     // popup
     else {
         Qt::WindowFlags flags(
+#ifdef Q_OS_MAC
```

Comment:
@cbjeukendrup do we really need this change on windows? It works perfectly fine there. So it should be:

```
#if defined(Q_OS_MAC) || defined(Q_OS_WIN)
    Qt::Tool
#else
    Qt::Popup // Popups can't be Qt::Tool on Linux Wayland, or they can't be relatvely positioned.
#endif
```